src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.cxx

```diff
@@ -26,8 +26,11 @@
 
 void
 t8_geometry_squared_disk::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords,
-                                            double out_coords[3]) const
+                                            const size_t num_coords, double out_coords[3]) const
 {
+  if (num_coords != 1)
+    SC_ABORT ("Error: Batch computation of geometry not yet supported.");
```

Comment:
I'll implement this once this PR is merged.

Reply:
Nice :)